apps/src/music/MusicView.jsx

```diff
@@ -295,6 +286,35 @@ class UnconnectedMusicView extends React.Component {
     });
   };
 
+  clearCode = () => {
+    // Default code.
+    const defaultCode = {
```

Comment:
(may be too nit-picky) can we extract out the default code into a helper file?

Reply:
Good call.  I figure this might come from levelbuilder eventually, but this was quite straightforward for now.